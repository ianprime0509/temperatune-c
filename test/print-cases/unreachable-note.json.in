{
  "name": "Note offset can't be calculated",
  "octaveBaseName": "C",
  "referencePitch": 440,
  "referenceName": "A",
  "referenceOctave": 4,
  "notes": {
    "A": ["A", 0],
    "C": ["A", 500],
    "E": ["E", 5]
  }
}
