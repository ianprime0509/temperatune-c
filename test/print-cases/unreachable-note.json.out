temperatune: no offset determined for note 'E'
